client/scala/scala-armada-client/src/test/scala/ArmadaClientSuite.scala

```diff
@@ -115,7 +138,11 @@ private class SubmitMockServer(jobMap: ConcurrentHashMap[String, Job], queueMap:
   }
 }
 
-private class JobsMockServer(jobMap: ConcurrentHashMap[String, Job]) extends JobsGrpc.Jobs {
+private class JobsMockServer(
+  jobMap: TrieMap[String, Job],
```

Comment:
this map doesn't appear to get used in this class.

Reply:
Good catch - this was from an earlier version where I envisioned tracking job details, etc, but that is definitey not used currently - fixed in commit ffdc1e18.

Reply:
i didn't catch it, intellij did